hacking_testing/minetest_env.py

```diff
@@ -392,6 +392,8 @@ def _write_config(self):
             config_file.write("mute_sound = true\n")
             config_file.write("show_debug = false\n")
             config_file.write("enable_client_modding = true\n")
+            config_file.write("video_driver = null\n")
```

Comment:
I think these config changes shouldn't be included because I was just adding them to get the test loop to run in colab. but then it turned out that only empty frames were sent from minetest.